code/modules/mob/living/burgerchat.dm

```diff
@@ -0,0 +1,87 @@
+// Thanks to Burger from Burgerstation for the foundation for this
+
+/* 
+BYOND Forum posts that helped me:
+http://www.byond.com/forum/post/1133166
+http://www.byond.com/forum/post/1072433
+http://www.byond.com/forum/post/940994
+http://www.byond.com/docs/ref/skinparams.html#Fonts
+*/
+
+/atom
```

Comment:
Does this *really* need to be defined on the `/atom/` level

That's a *lot* of extra lists for something most things won't utilize.

Reply:
I just checked and this can definitely be at the `/mob/living` level. Or `/mob` level depending on if you change things based on my other review comment.